docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -76,6 +76,18 @@ https://raw.githubusercontent.com/pulp/pulp/pulp-2.8.0-1/playpen/mongoengine/REA
 Upgrade Instructions for 2.7.x --> 2.8.x
 ----------------------------------------
 
+.. note::
+    When upgrading on CentOS7, it is possible that your system has 'python-semantic-version'
+    package installed. As result you will experience a problem when updating Pulp packages. If this
+    package is present on your system you should remove it::
+
+        sudo yum remove python-semantic-version
+
+    This will remove a number of pulp packages also. You will install the new packages::
+
+        sudo yum update
+        sudo yum groupinstall pulp-server-qpid
```

Comment:
One more thought - perhaps we could put this in for now, and then have another release ready to go that solves it some technical way.

I'll LGTM it since it's affecting current users now. We can take the time to solve it another way next if we want to.